### ISSUE TITLE
loop_var gets the whole list instead of item in list with python 3.8

### ISSUE THREAD

**Issue by pankajrlal** [ansible/ansible#74615]
### Summary

I am using loop, loop_control and loop_var. To loop I am passing a variable that is a list of items. the loop_var is assigned the name of an item in the list when working with python 3.6, but gets the whole list assigned when working with python 3.8

### Issue Type

Bug Report

### Component Name

ansible-playbook

### Ansible Version

```console
$ ansible --version
ansible 2.5.5
  config file = /home/pankaj/mammoth/appspace/starter-kit/ansible.cfg
  configured module search path = ['/home/pankaj/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/pankaj/mammoth/appspace/opsenv/lib/python3.8/site-packages/ansible
  executable location = /home/pankaj/mammoth/appspace/opsenv/bin/ansible
  python version = 3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]
```

### Configuration

```console
$ ansible-config dump --only-changed
ALLOW_WORLD_READABLE_TMPFILES(/home/pankaj/mammoth/appspace/starter-kit/ansible.cfg) = True
DEFAULT_FILTER_PLUGIN_PATH(/home/pankaj/mammoth/appspace/starter-kit/ansible.cfg) = ['/home/pankaj/mammoth/appspace/operations/playbooks/filter_plugins']
DEFAULT_REMOTE_USER(/home/pankaj/mammoth/appspace/starter-kit/ansible.cfg) = vagrant
DEFAULT_ROLES_PATH(/home/pankaj/mammoth/appspace/starter-kit/ansible.cfg) = ['/home/pankaj/mammoth/appspace/operations/playbooks/roles']
HOST_KEY_CHECKING(/home/pankaj/mammoth/appspace/starter-kit/ansible.cfg) = False
```

### OS / Environment

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

### Steps to Reproduce

```yaml (paste below)

```

[test.txt](https://github.com/ansible/ansible/files/6440221/test.txt)
[included.txt](https://github.com/ansible/ansible/files/6440222/included.txt)
[config.txt](https://github.com/ansible/ansible/files/6440223/config.txt)

### Expected Results

I expected the loop_var to get value of each item per loop iteration. However it got the whole list and loop ran only once. this works as expected with python 3.6 but not with python 3.8

### Actual Results

```console
[WARNING]: Unable to parse /etc/ansible/hosts as an inventory source

 [WARNING]: No inventory was parsed, only implicit localhost is available

 [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

[DEPRECATION WARNING]: profile_tasks callback, does not support setting 'options', it will work for now,  but this will be required in the future and should be updated,  see the 2.4 porting guide for 
details.. This feature will be removed in version 2.9. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.

PLAY [Sample file to show problem] ****************************************************************************************************************************************************************************

TASK [Gathering Facts] ****************************************************************************************************************************************************************************************
ok: [localhost]

TASK [include tasks] ******************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"msg": "Invalid data passed to 'loop', it requires a list, got this instead: ['A', 'B']. Hint: If you passed a list/dict of just one element, try adding wantlist=True to your lookup invocation or use q/query instead of lookup."}
	to retry, use: --limit @/home/pankaj/mammoth/appspace/operations/playbooks/test.retry

PLAY RECAP ****************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1   

TASK: Gathering Facts --------------------------------------------------- 0.81s
TASK: include tasks ----------------------------------------------------- 0.02s
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by ansibot** [comment]
Files identified in the description:
* `lib/ansible/playbook`

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by mkrizek** [comment]
I cannot reproduce the issue you are seeing. You are using unsupported version of Ansible, please upgrade to a [supported version](https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#ansible-core-release-cycle) (2.9+).

needs_info

**Comment by mkrizek** [comment]
I am going to go ahead and close this per above. If the issue persists with a recent version of Ansible, please open a new issue.

If you have further questions please stop by IRC or the mailing list:
* IRC: #ansible on irc.freenode.net
* mailing list: https://groups.google.com/forum/#!forum/ansible-project
